sdk/keyvault/keyvault-certificates/test/utils/testClient.ts

```diff
@@ -6,6 +6,7 @@ import { PollerLike, PollOperationState } from "@azure/core-lro";
 import { operationOptionsToRequestOptionsBase } from "@azure/core-http";
 import { RestoreCertificateBackupPoller } from "./lro/restore/poller";
 import { BeginRestoreCertificateBackupOptions } from "./lro/restore/operation";
+import { testPollerProperties } from "./recorderUtils";
```

Comment:
I assume this object has the right values for polling for each test mode.

Reply:
Correct! https://github.com/Azure/azure-sdk-for-js/blob/ab612a816be2b71e8c33280a298b1cd6b475aa10/sdk/keyvault/keyvault-certificates/test/utils/recorderUtils.ts#L20-L22